### PR TITLE
OCD-3474: AQA tests for listing- Reject

### PR DESCRIPTION
**Pull request by palakpatel311**

### CODE REVIEW

File: e2e/pages/administration/confirm/confirm.spec.js [hunk 1]
```diff
@@ -6,25 +6,53 @@ import LoginComponent from '../../../components/login/login.po';
...
 
-describe('Confirm page', () => {
+describe('when user is on confirm listing page', () => {
+    beforeEach(() => {
+        uploadPage = new UploadPage();
+        confirmPage = new ConfirmPage();
+        loginComponent = new LoginComponent();
+        hooks = new Hooks();
+        hooks.open('#/administration/upload');
+        loginComponent.loginAsACB();
```
Comment (andlar) [inline e2e/pages/administration/confirm/confirm.spec.js]: The "loginComponent.logIn('acb')" method should be in staging now, which means you ought to be able to use that here so we can work to remove this `loginAsACB` method

**Review by kekey1 — DISMISSED** [review]
Nothing to add beyond Andrew's comment

File: e2e/pages/administration/confirm/confirm.spec.js (hunk 1 above)
Comment (palakpatel311) [reply e2e/pages/administration/confirm/confirm.spec.js]: Done

Comment (andlar) [reply e2e/pages/administration/confirm/confirm.spec.js]: I don't see this as having changed

Comment (palakpatel311) [reply e2e/pages/administration/confirm/confirm.spec.js]: done

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -3202,9 +3202,15 @@ chrome-trace-event@^1.0.2:
     tslib "^1.9.0"
 
 chromedriver@^85.0.0:
+<<<<<<< HEAD
```
Comment (kekey1) [inline yarn.lock]: this still looks like a conflict

**Review by kekey1 — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (palakpatel311) [reply yarn.lock]: done

**Review by kekey1 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request merged**